DOCKER-README.md

```diff
@@ -3,39 +3,65 @@
 ## WARNING
 
 > [!WARNING]
-> Currently Docker cannot be used to run Quilibrium.
+> The Quilibrium docker container requires host configuration changes.
 
 There are extreme buffering requirements, especially during sync, and these in turn require `sysctl`
-configuration changes that unfortunately are not supported by Docker.
+configuration changes that unfortunately are not supported by Docker. But if these changes are made on
+the host machine, then luckily containers seem to automatically have the larger buffers.
 
 The buffer related `sysctl` settings are `net.core.rmem_max` and `net.core.wmem_max` and they both
 should be set to `600,000,000` bytes. This value allows pre-buffering of the entire maximum payload
 for sync.
 
+You can tell that the buffer size is not large enough by noticing this log entry at beginning when 
+Quilibrium starts, a few lines below the large logo:
+> failed to sufficiently increase receive buffer size (was: 208 kiB, wanted: 2048 kiB, got: 416 kiB).
+> See https://github.com/quic-go/quic-go/wiki/UDP-Buffer-Sizes for details.
+
 To read the currently set values:
 ```shell
 sysctl -n net.core.rmem_max
 sysctl -n net.core.wmem_max
 ```
 
-To set new values:
+To set new values, this is not a persistent change:
 ```shell
 sudo sysctl -w net.core.rmem_max=600000000
 sudo sysctl -w net.core.wmem_max=600000000
 ```
 
+To persistently set the new values add a configuration file named `20-quilibrium.conf` to
+`/etc/sysctl.d/`. The file content should be:
+```
+# Quilibrium buffering requirements, especially during sync.
+# The value could be as low as 26214400, but everything would be slower.
+
+net.core.rmem_max = 600000000
+net.core.wmem_max = 600000000
+```
+
+
 ## Build
 
-Build the docker image:
+In the repository root folder, where the [Dockerfile](Dockerfile) file is, build the docker image:
 ```shell
-docker build -t quilibrium -t quilibrium:1.2.9 .
+docker build --build-arg GIT_COMMIT=$(git log -1 --format=%h) -t quilibrium -t quilibrium:1.2.15 .
 ```
 
-Use latest version instead of `1.2.9`.
+Use latest version instead of `1.2.15`.
 
 
 ## Run
 
+You can run Quilibrium on the same machine where you built the image, from the same repository root
+folder where [docker-compose.yml](docker-compose.yml) is.
+
+You can also copy `docker-compose.yml` to a new folder on a server and run it there. In this case you
+have to have a way to push your image to a Docker image repo and then pull that image on the server.
+Github offers such an image repo and a way to push and pull images using special authentication
+tokens. See
+[Working with the Container registry](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-container-registry).
```

Comment:
@CassOnMars and plans to start pushing images to Docker Hub? Happy to help with that, let me know.

FYI working on building a much leaner image, in a followup PR.

Reply:
no immediate plans but if you want to take point on this i'm supportive!